docs/intro/what-is-zap.md

```diff
@@ -0,0 +1,45 @@
+# What is Zap?
+
+Zap is a tool to generate highly efficient networking code for Roblox. It has no compromises on performance or developer experience. Zap takes a description of your data and event, and generates code specific to your game.
+
+<div class="custom-block tip" style="padding-top: 8px">
```

Comment:
Why did you add a custom tip? It may look better if you did an info tip with a custom title

Reply:
I didn't really want to give it a title, it looked kinda clunky.